### PR TITLE
Oldhippiehair

### PR DESCRIPTION
**Pull request by ghost**
:cl: QuarianCommando
add: Re-added the old Hippie hairs
/:cl:

Re-added old hippie hairs, so the autists don't shit themselves.

### CODE REVIEW

**Comment by ghost** [comment]
Motherfucker this is sloppy, but it should be functional.

File: hippiestation.dme [hunk 1]
```diff
@@ -1419,7 +1419,6 @@
 #include "code\modules\mob\dead\new_player\new_player.dm"
 #include "code\modules\mob\dead\new_player\poll.dm"
 #include "code\modules\mob\dead\new_player\preferences_setup.dm"
-#include "code\modules\mob\dead\new_player\sprite_accessories.dm"
```
Comment (j-mie) [inline hippiestation.dme:1422]: Don't do this

File: hippiestation.dme [hunk 2]
```diff
@@ -2081,6 +2080,7 @@
 #include "hippiestation\code\modules\clothing\under.dm"
 #include "hippiestation\code\modules\hydroponics\buttflower.dm"
 #include "hippiestation\code\modules\hydroponics\limbflower.dm"
+#include "hippiestation\code\modules\mob\dead\new_player\sprite_accessories.dm"
```
Comment (j-mie) [inline hippiestation.dme:2083]: Keep this

**Review by tortellinitony — CHANGES_REQUESTED** [review]
You can just add new sprite accessories...in the file we already have.

File: hippiestation/code/modules/mob/dead/new_player/sprite_accessories.dm [hunk 3]
```diff
@@ -0,0 +1,1423 @@
+/*
```
Comment (j-mie) [inline hippiestation/code/modules/mob/dead/new_player/sprite_accessories.dm:1]: Just add the unique hairs to this file, have a new hair type:

```/datum/sprite_accessory/hair/hippie/
    icon = 'hippiestation/icons/mob/human_face.dmi'	  // default icon for all hairs
```

then add all of our hair as this:
```
/datum/sprite_accessory/hair/hippie/twincurl_alt
```

if that doesn't work - just remove the hippie bit and add the icon to each of the hippie hair

**Review by j-mie — CHANGES_REQUESTED** [review]

**Pull request closed without merging**